Automattic/pocket-casts-ios
# Fix #2132: Fix iPad Crash when presenting popover from share button

| 📘 Part of: #1910 |
|:---:|

Fixes #2132

The popover `sourceRect` was missing and this adds it back in a bit of a hacky way.
Ideally we would switch to the SwiftUI native `ShareLink` view but that comes with a host of challenges with the `Transferable` conformance. When I tested it before, many apps did not accept certain transferable data representations so I'd prefer to continue with the `UIActivityViewController` method for now.

![CleanShot 2024-09-04 at 09 27 24@2x](https://github.com/user-attachments/assets/9fce3dac-a1ee-4344-a78f-9034bf04c0f6)

## To test

* Open an Episode on iPad
* Tap the Share button
* Tap the "More" button
* Verify that a popover appears with sharing options

## Checklist
